src/app/shared/components/list/generic-list.component.ts

```diff
@@ -0,0 +1,15 @@
+import {Component, Input, TemplateRef} from '@angular/core';
+import {NgClass, NgTemplateOutlet} from '@angular/common';
+
+@Component({
+  selector: 'generic-list',
+  standalone: true,
+  imports: [NgTemplateOutlet, NgClass],
+  templateUrl: './generic-list.component.html',
+  styleUrl: './generic-list.component.scss',
+})
+export class GenericListComponent {
```

Comment:
Since you're using `<ul>`, this is not _that_ generic, it's only a `GenericUnorderedList` - there are also other lists, however.

Reply:
I renamed it accordingly